subs2srs.lua

```diff
@@ -342,13 +351,13 @@ local function export_to_anki(gui)
         sub['text'] = string.format("mpvacious wasn't able to grab subtitles (%s)", os.time())
     end
     local snapshot_timestamp = mp.get_property_number("time-pos", 0)
-    local snapshot_filename = filename_factory.make_snapshot_filename(snapshot_timestamp, config.snapshot_extension)
-    local audio_filename = filename_factory.make_audio_filename(sub['start'], sub['end'], config.audio_extension)
+    local video_filename = encoder.video.make_filename(sub['start'], sub['end'], snapshot_timestamp)
+    local audio_filename = encoder.audio.make_filename(sub['start'], sub['end'])
 
-    encoder.create_snapshot(snapshot_timestamp, snapshot_filename)
-    encoder.create_audio(sub['start'], sub['end'], audio_filename, audio_padding())
+    encoder.video.create(sub['start'], sub['end'], snapshot_timestamp, video_filename)
+    encoder.audio.create(sub['start'], sub['end'], audio_filename, audio_padding())
 
-    local note_fields = construct_note_fields(sub['text'], sub['secondary'], snapshot_filename, audio_filename)
+    local note_fields = construct_note_fields(sub['text'], sub['secondary'], video_filename, audio_filename)
 
```

Comment:
Technically you're not making any *videos* so `video_filename` and ` encoder.video` are really just `shapshot_filename` and `encoder.snapshot`.

Reply:
Renamed it